node_modules/oae-core/topnavigation/js/topnavigation.js

```diff
@@ -67,7 +67,8 @@ define(['jquery', 'oae.core', 'activityadapter'], function($, oae, ActivityAdapt
         var initLeftMenu = function() {
```

Comment:
It could have been a feature that was lost with between topnav refactoring and inviting guests. But I'm not entirely sure.